app/src/components/SoldArtworkList.tsx

```diff
@@ -36,26 +36,25 @@ class SoldArtworkList extends React.Component<ContractProps, SoldArtworkListStat
     const governance = this.props.contracts.Governance;
     const currentAccount = this.props.accounts[0];
     let eventsFound = 0;
-    const soldInfoArray: any[] = [];
+    const soldInfoArray: SoldInformation[] = [];
+    const filter = { filter: { from: currentAccount } };
+    const options = { filter, fromBlock: 0 };
 
     await governance.getPastEvents(
-      'RecordARR',
-      (errors: any, events: any) => {
-        if (!errors) {
-          for (let i = 0; i < events.length; i++) {
-            if (events[i].returnValues.from === currentAccount) {
-              eventsFound += 1;
-              const soldInfo: SoldInformation = {
-                tokenId: events[i].returnValues.tokenId,
-                price: events[i].returnValues.price,
-                newOwner: events[i].returnValues.to,
-              };
-              soldInfoArray.push(soldInfo);
-            }
-          }
-        }
-      },
-    );
+      'RecordARR', options).then(function (events: any) {
+      eventsFound = events.length;
+      for (let i = 0; i < events.length; i++) {
+        const soldInfo: SoldInformation = {
+          tokenId: events[i].returnValues.tokenId,
+          price: events[i].returnValues.price,
+          newOwner: events[i].returnValues.to,
+        };
+        soldInfoArray.push(soldInfo);
+      }
+    }).catch(function (error: any) {
+      alert(error.message);
```

Comment:
would you mind doing this in a console.log instead? otherwise it is the only instance of an 'alert' for an error.

i think you could even just do 
`.catch(console.log)`

Reply:
Pretty sure Dan had an alert somewhere too but sure, will change